### ISSUE TITLE
Irragular nuclio function name on kubernetes

### ISSUE THREAD

**Issue by xuchenCN** [openvinotoolkit/cvat#2393]
I try to deploy nulcio funcions on kubernetes via [deply.sh](https://github.com/openvinotoolkit/cvat/blob/develop/serverless/deploy.sh)

But it will occur the DNS-1123 error cause the function name in the function.yaml

```
Error - NuclioFunction.nuclio.io "openvino.omz.public.faster_rcnn_inception_v2_coco" is invalid: metadata.name: Invalid value: "openvino.omz.public.faster_rcnn_inception_v2_coco": a DNS-1123 subdomain must consist of lower case alphanumeric characters, '-' or '.', and must start and end with an alphanumeric character (e.g. 'example.com', regex used for validation is '[a-z0-9]([-a-z0-9]*[a-z0-9])?(\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*')
    /nuclio/pkg/platform/kube/deployer.go:100
```

**Comment by nmanovic** [comment]
@xuchenCN , could you please help us to fix the issue?

**Comment by xuchenCN** [comment]
@nmanovic fixed in PR #2415
